skills_taxonomy_v2/examples/Extract Skills.py

```diff
@@ -0,0 +1,93 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+# cd ../..
+
+# %%
+import json
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+# %%
+from skills_taxonomy_v2.examples.extract_skills import (
+    load_prerequisites, split_skills, predict_skill_sents, reduce_embeddings, get_skill_name, cluster_embeddings
+)
+from skills_taxonomy_v2.pipeline.skills_extraction.get_sentence_embeddings import get_embeddings
+
+# %%
+sent_classifier_model_dir = 'outputs/sentence_classifier/models/2021_08_16.pkl'
+job_adverts_file = 'skills_taxonomy_v2/examples/job_advert_examples.txt'
+
+# Parameters - these will need tweaking depending on your input data
+reduction_n_neighbors = 6
+reduction_min_dist = 0.0
+clustering_eps = 1
+clustering_min_samples = 1
+
+# %%
+nlp, bert_vectorizer, sent_classifier = load_prerequisites(sent_classifier_model_dir)
+
+# Load your job advert texts; a list of dicts with the keys "full_text" and "job_id"
+with open(job_adverts_file) as f:
+    job_adverts = json.load(f)
+
+# Run the pipeline to extract skills
```

Comment:
We could add the elements of the pipeline to a simple extract_skills function in extract_skills_utils.py